.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -36,12 +41,12 @@
 <!-- You can add additional checkboxes if it makes sense to only review parts of the code or functionality. -->
 <!-- When changes are pushed, uncheck the affected boxes. (Not all changes require full re-reviews.) -->
 
-- Code Review
-  - [ ] Review 1
-  - [ ] Review 2
-- Manual Tests
-  - [ ] Test 1
-  - [ ] Test 2
+#### Code Review
+- [ ] Review 1
+- [ ] Review 2 
+#### Manual Tests
+- [ ] Test 1
+- [ ] Test 2
 
 ### Test Coverage
 <!-- Please add the test coverage for all changes files here. You can see this when executing the tests locally (see build.gradle and package.json) or when looking into the corresponding Bamboo build plan -->
```

Comment:
As we are changing the template anyways, I think it would be a good idea to indicate, which coverage we should report. Line coverage I assume?